Spec/API.md

```diff
@@ -339,7 +339,7 @@ Request:
 ```
 GET /api/v1/discover/new[?page=<page>]
 ```
-Where `page` is the page offset into the result set (0 by default).
+Where `page` is the page offset into the result set (1 by default).
 
```

Comment:
Changed to match code, default page is 1. Thought it made more sense to start at 1 in this case.